[2025-02-06T08:59:21.427+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T08:59:21.446+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.task_random_choice scheduled__2025-01-01T01:26:30+00:00 [queued]>
[2025-02-06T08:59:21.458+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.task_random_choice scheduled__2025-01-01T01:26:30+00:00 [queued]>
[2025-02-06T08:59:21.459+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T08:59:21.496+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): task_random_choice> on 2025-01-01 01:26:30+00:00
[2025-02-06T08:59:21.499+0000] {standard_task_runner.py:72} INFO - Started process 1265 to run task
[2025-02-06T08:59:21.501+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'my_first_dag', 'task_random_choice', 'scheduled__2025-01-01T01:26:30+00:00', '--job-id', '294', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp77lumd4g']
[2025-02-06T08:59:21.502+0000] {standard_task_runner.py:105} INFO - Job 294: Subtask task_random_choice
[2025-02-06T08:59:21.589+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.task_random_choice scheduled__2025-01-01T01:26:30+00:00 [running]> on host 128ac2960630
[2025-02-06T08:59:21.716+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='task_random_choice' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T01:26:30+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T01:26:30+00:00'
[2025-02-06T08:59:21.717+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-06T08:59:21.718+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-06T08:59:21.718+0000] {logging_mixin.py:190} INFO - Current task name:task_random_choice state:running start_date:2025-02-06 08:59:21.447116+00:00
[2025-02-06T08:59:21.718+0000] {logging_mixin.py:190} INFO - Dag name:my_first_dag and current dag run status:running
[2025-02-06T08:59:21.719+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T08:59:21.719+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-06T08:59:21.720+0000] {python.py:240} INFO - Done. Returned value was: task_2
[2025-02-06T08:59:21.721+0000] {branch.py:38} INFO - Branch into task_2
[2025-02-06T08:59:21.721+0000] {skipmixin.py:230} INFO - Following branch ('task_2',)
[2025-02-06T08:59:21.735+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-06T08:59:21.797+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T08:59:21.798+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=task_random_choice, run_id=scheduled__2025-01-01T01:26:30+00:00, execution_date=20250101T012630, start_date=20250206T085921, end_date=20250206T085921
[2025-02-06T08:59:21.834+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-06T08:59:21.835+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-06T08:59:21.835+0000] {logging_mixin.py:190} INFO - Dag name:my_first_dag queued_at:2025-02-06 08:59:15.280968+00:00
[2025-02-06T08:59:21.836+0000] {logging_mixin.py:190} INFO - Task hostname:128ac2960630 operator:BranchPythonOperator
[2025-02-06T08:59:21.875+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T08:59:21.925+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T08:59:21.936+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
